src/coreclr/vm/jitinterface.h

```diff
@@ -103,6 +103,10 @@ BOOL LoadDynamicInfoEntry(Module *currentModule,
 
 #endif // TARGET_X86
 
+// These must be implemented by the JIT in order to support the "jithelpers" functionality.
```

Comment:
I am not following what this comment is trying to say

Reply:
No idea what I was thinking there.

I've replaced with
`// These must be implemented in assembly and generate a TransitionBlock then calling JIT_PatchpointWorkerWithPolicy in order to actually be used.`